modules/dists/StencilDist.chpl

```diff
@@ -1531,6 +1464,7 @@ iter StencilArr.dsiBoundaries(param tag : iterKind) where tag == iterKind.standa
 //
 // Copies the range slice function in ChapelArray.
 //
+pragma "no copy return"
```

Comment:
(I mostly mean eventually, it probably doesn't make sense now b/c the view might not be distributed right)